config/locales/cmn/i18n.rb

```diff
@@ -3,7 +3,7 @@
 { cmn: {
   i18n: {
     dir: 'ltr',
-    iso2: 'zh',
+    iso2: 'zh-cn',
```

Comment:
Why do you change Chinese `ISO2` code (internal code for translation for _ISO639 code in 2 characters_)?
`zh` is the official code (https://en.wikipedia.org/wiki/Chinese_language). Do you encountered any trouble with JS i18n lib? Instead of ISO2, we may try to change reference code to IETF one which best suit here.
In all case, `iso2` code is expected to be on 2 characters only.

Reply:
If nothing blocking with 2-characters code, only revert this change please.